### DISCUSSION TITLE
Overview of guides structure and sections

### DISCUSSION THREAD

**Discussion by jurra** [TU-Delft-DCC/TU-Delft-DCC.github.io#184] [Solutions]
Here a couple of suggestions and definitions we have agreed upon in our guides meeting:
- Having a getting started guide with an introduction on how to approach Computing Resources (Infrastructure), Data Management and Research Software. Examples I have found that are relevant:
- https://scicomp.aalto.fi/scicomp/zen-of-scicomp/
- https://sebranchett.github.io/TUDlife/Support-for-Research-Computing
- https://shiftmag.dev/code-quality-good-enough-2034/

I am not sure if the right word is getting started at the moment, because I have been reviewing getting started, and normally is in the context of using a tool and achieving a result. It is very common in packages, programming languages, or technology usage in general.

We agree that each section has elements of planning, infrastructure and skills among other subjects.
For example infrastructure for data management is different from infrastructure for software management (github, gitlab, etc.) And infrastructure for computing entails HPC, cloud, etc. The same goes with tools, skills and so on.

Counter example of something we currently have: Gitlab is now under Computing Infrastructure, it should be under the Research Software section instead.

**Comment by rortizmerino** [comment]
Building up from a discussion with @edvinskis, and from @Selkubi discussion on [Structuring of the Data Management Section](https://github.com/TU-Delft-DCC/TU-Delft-DCC.github.io/discussions/180), would be great to consider the Data and Software life cycles.

Say, the data life cycle consists of the following stages:

1. Plan
2. Collect
3. Process
4. Analyse
5. Preserve
6. Share
7. Reuse

Then, say the software life cycle consists of the following stages:

1. Plan
2. Analyse
3. Design
4. Implementation
5. Testing 
6. Integration
7. Maintenance

We could have a getting started section explaining those cycles an their stages, then have the guides' sections organized accordingly.

We might have to agree on which stages and definitions to use as there's loads out there. Perhaps there's already a definition provided by TU Delft? that'd be perfect.
